packages/client/test/helpers.ts

```diff
@@ -0,0 +1,89 @@
+import { Operation, resource, main } from 'effection';
+import { Channel } from '@effection/channel';
+import { Subscribable } from '@effection/subscription';
+import { express, Socket } from '@bigtest/effection-express';
+import { Mailbox } from '@bigtest/effection';
```

Comment:
if `Mailbox` is a pattern we don't want to use anymore should we use something else?

Unless I have got this wrong about `Mailbox` being deceased.

Reply:
@dagda1 Absolutely agree. My goal with this PR is to extract the client as seamlessly as possible, wrap a test suite around it (which it didn't have before). 

Now that those are in place, we can refactor it internally in order to remove Mailbox and go with a `Subscription` based approach in follow-on PR's